packages/server/src/api/routes/datasource.ts

```diff
@@ -20,8 +20,8 @@ router
     authorized(permissions.BUILDER),
     datasourceController.verify
   )
-  .get(
-    "/api/datasources/:datasourceId/info",
+  .post(
```

Comment:
Shouldn't this be kept as a get?

Reply:
No, we need a post body for this - so we can send up the datasource contents.